diff --git a/embodied_gen/data/backproject_v2.py b/embodied_gen/data/backproject_v2.py
index 92c0a76..d2f8a4e 100644
--- a/embodied_gen/data/backproject_v2.py
+++ b/embodied_gen/data/backproject_v2.py
@@ -40,7 +40,6 @@ from embodied_gen.data.utils import (
     post_process_texture,
     save_mesh_with_mtl,
 )
-from embodied_gen.models.delight_model import DelightingModel
 from embodied_gen.models.sr_model import ImageRealESRGAN
 from embodied_gen.utils.process_media import vcat_pil_images
 
@@ -781,7 +780,7 @@ def parse_args():
 
 
 def entrypoint(
-    delight_model: DelightingModel = None,
+    delight_model: "DelightingModel | None" = None,
     imagesr_model: ImageRealESRGAN = None,
     **kwargs,
 ) -> trimesh.Trimesh:
@@ -812,6 +811,8 @@ def entrypoint(
 
     args.color_path = as_list(args.color_path)
     if args.delight and delight_model is None:
+        from embodied_gen.models.delight_model import DelightingModel
+
         delight_model = DelightingModel()
 
     color_grid = [Image.open(color_path) for color_path in args.color_path]
