diff --git a/P3-SAM/model.py b/P3-SAM/model.py
index 15d64d1..dc4459e 100644
--- a/P3-SAM/model.py
+++ b/P3-SAM/model.py
@@ -15,7 +15,12 @@ The model is composed of three parts:
 '''
 def build_P3SAM(self): #build p3sam
     ######################## Sonata ########################
-    self.sonata = sonata.load("sonata", repo_id="facebook/sonata", download_root='/root/sonata')
+    self.sonata = sonata.load(
+        "sonata",
+        repo_id="facebook/sonata",
+        download_root="/root/sonata",
+        custom_config={"enable_flash": False},
+    )
     self.mlp = nn.Sequential(
             nn.Linear(1232, 512),
             nn.GELU(),
