diff --git a/embodied_gen/utils/monkey_patch/p3sam.py b/embodied_gen/utils/monkey_patch/p3sam.py
index 0d6d719..46828ce 100644
--- a/embodied_gen/utils/monkey_patch/p3sam.py
+++ b/embodied_gen/utils/monkey_patch/p3sam.py
@@ -31,6 +31,7 @@ def monkey_patch_p3sam():
             download_root=os.path.expanduser(
                 "~/.cache/huggingface/hub/sonata"
             ),
+            custom_config={"enable_flash": False},
         )
         self.mlp = nn.Sequential(
             nn.Linear(1232, 512),
